releases/sandbox/gp-pyspark/gp-pyspark-canary.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: gp-pyspark
+  namespace: gp-pyspark
+  annotations:
+    fluxcd.io/automated: "true"
+spec:
+  releaseName: gp-pyspark
+  chart:
+    git: git@github.com:Git-Prime/gphelm-chart-repo.git
+    path: charts/gp-pyspark/v0.0.1/
+    version: 0.0.1
+    ref: master
+  values:
+    jobs:
+    - name: gp-pyspark-canary-nightly
+      image:
```

Comment:
I think we should start calling the jobs as `gp-pyspark-canary-full` and `gp-pyspark-canary-partial`. Because we'll be running the full job every night as it looks like it is feasible, and suspend the partial job